dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -493,11 +504,12 @@ def confirm_upgrade_account
     SecureRandom.alphanumeric length
   end
 
-  private def log_unauthorized(exception, context = nil)
-    Honeybadger.notify(
-      exception,
-      context: context
-    )
+  private def log_metric(event, attributes = {})
```

Comment:
This helper method initially did more, now it only wraps `LtiLogger.log_event`. Considering removing it and just using `LtiLogger.log_event` directly in this controller instead.